### DISCUSSION TITLE
Integration with CAD

### DISCUSSION THREAD

**Discussion by PabloMBarral** [oemof/tespy#445] [General]
Hi, for those willing to implement TESPy in an engineering studio environment, turns out that

https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-update-AutoCAD-block-attributes-using-Microsoft-Excel.html

is an excellent choice to link the dataframe output from TESPy with CAD.

All that's needed is to export the states from the dataframe to Excel, and then paste it when you edit the .txt with the attributes. This step could be automatized as well.

This is useful when you need to document several process conditions in a project (a CHP plant, for example).

Regards,

**Comment by fwitte** [comment]
@PabloMBarral: thanks for the tip!
